<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1117</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1049</width>
    <height>610</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pattaya RAT Panel - $VERSION [ Online bots: $ONLINE_BOT ] [ User: $PANEL_USER ]</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/assets/images/rat.png</normaloff>:/assets/images/rat.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="bot_table_view">
      <property name="minimumSize">
       <size>
        <width>991</width>
        <height>371</height>
       </size>
      </property>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
    <item>
     <widget class="QTabWidget" name="control_tab">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Panel log viewer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTextBrowser" name="panel_log_text_browser"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Server log viewer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="server_log_text_browser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1117</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPattaya">
    <property name="title">
     <string>Pattaya</string>
    </property>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionRefresh_Bot"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Setting</string>
    </property>
    <addaction name="actionServer_setting"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Pattaya_Project"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionDark"/>
    <addaction name="actionLight"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionFile_to_Base64_Encode"/>
   </widget>
   <addaction name="menuPattaya"/>
   <addaction name="menuSetting"/>
   <addaction name="menuTheme"/>
   <addaction name="menuTools"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
   <addaction name="actionRefresh_Bot"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionServer_setting"/>
   <addaction name="actionBot_Terminal"/>
   <addaction name="separator"/>
   <addaction name="actionFile_to_Base64_Encode"/>
  </widget>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/plug-connect.png</normaloff>:/assets/images/plug-connect.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/cross.png</normaloff>:/assets/images/cross.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionServer_setting">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/server--pencil.png</normaloff>:/assets/images/server--pencil.png</iconset>
   </property>
   <property name="text">
    <string>Server setting</string>
   </property>
  </action>
  <action name="actionAbout_Pattaya_Project">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/information-button.png</normaloff>:/assets/images/information-button.png</iconset>
   </property>
   <property name="text">
    <string>About Pattaya Project</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/qt.png</normaloff>:/assets/images/qt.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/plug-disconnect.png</normaloff>:/assets/images/plug-disconnect.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/monitor-off.png</normaloff>:/assets/images/monitor-off.png</iconset>
   </property>
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/monitor.png</normaloff>:/assets/images/monitor.png</iconset>
   </property>
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionRefresh_Bot">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/reload.png</normaloff>:/assets/images/reload.png</iconset>
   </property>
   <property name="text">
    <string>Refresh Bot</string>
   </property>
  </action>
  <action name="actionBot_Terminal">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/terminal.png</normaloff>:/assets/images/terminal.png</iconset>
   </property>
   <property name="text">
    <string>Bot Terminal</string>
   </property>
   <property name="toolTip">
    <string>Bot Terminal</string>
   </property>
  </action>
  <action name="actionFile_to_Base64_Encode">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/assets/images/edit-language.png</normaloff>:/assets/images/edit-language.png</iconset>
   </property>
   <property name="text">
    <string>File to Base64 Encode</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
